docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,255 @@
+# Earth Observation guide
+
+This guide aims to help users who wish to work with Earth Observation (EO) data using CSC's computing resources. The purpose of this guide is to help you find the right data and tools for your raster data based EO tasks. If you are interested in the funamentals of EO, please check the resources at the bottom of the page first.
+
+**What are the benefits of using EO data**
+
+* Possibility to observe wide area at same time
+* Non-intrusive
+* Same sensor for different parts of the world
+* Time series 
+
+**EO data in raster format**
```

Comment:
I find this still rather strange here. I would rather find some external place where rasters are explained, for most readers should be known already.

Reply:
moved to box